rsa-cryptosystem-signature/index02.js

```diff
@@ -0,0 +1,6 @@
+/*
+The fixed-size value is also known as a hash value.
```

Comment:
I thought of the same approach but it'll be a kinda spoon-feeding. This lesson will come after the JavaScript lesson so, we should let campers figure out where the code should go and I want them to make some mistakes ;)

Reply:
@vkWeb, yes, I know what you mean. We definitely want to avoid spoon feeding learners the answers! However the format will be to position the test descriptions near the line that needs to be changed. This should help us keep the text as minimal as possible. Don't worry, there are still many good ways to challenge learners as they work through this, like with the modulo operator mentioned below.

Reply:
Okay. I'll shift descriptions to their appropriate places.